qiskit_ibm_runtime/base_primitive.py

```diff
@@ -154,16 +169,30 @@ def _run_primitive(self, primitive_inputs: Dict, user_kwargs: Dict) -> RuntimeJo
 
         combined = Options._set_default_resilience_options(combined)
         combined = Options._remove_none_values(combined)
-
         primitive_inputs.update(Options._get_program_inputs(combined))
 
-        if self._backend and combined["transpilation"]["skip_transpilation"]:
+        if self._backend and combined["transpilation"]["skip_transpilation"] and not run_simulator:
             for circ in primitive_inputs["circuits"]:
                 self._backend.check_faulty(circ)
 
         logger.info("Submitting job using options %s", combined)
 
         runtime_options = Options._get_runtime_options(combined)
+
+        if run_simulator:
+            # do we need to keep the other 'flat' options as well,
+            # for passing to terra directly?
+            primitive_inputs["optimization_level"] = combined["optimization_level"]
+            runtime_options["backend"] = self._backend
+
+            return self._service.run(
+                program_id=self._program_id(),
+                options=runtime_options,
+                inputs=primitive_inputs,
+                # are the following relevant for simulators?
+                # callback=combined.get("environment", {}).get("callback", None),
+                # result_decoder=DEFAULT_DECODERS.get(self._program_id()),
```

Comment:
I think we can leave these out for simulators

Reply:
In terms of the other options, I think it'd be nice to have the run options like:

`{'shots': 400, 'init_qubits': True, 'noise_model': None, 'seed_simulator': None}`
and 
`{'max_execution_time': None, 'log_level': 'WARNING', 'job_tags': []}`

Reply:
- `shots` and `seed_simulator` are already supported.
- I couldn't fine `init_qubits` or anything similar in the `AerSimulator`, so I don't think there is any point of passing it on. Let me know if you know otherwise. I did find it as a parameter of `assemble_circuits` in Terra, but I am not sure how and if this is called.
- Same as above for `max_execution_time`.
- `noise_model` - I think this is only relevant when calling `AerSimulator` and not for fake backends, because the latter define their own noise models. For Aer, I pass along all the options that are `AerSimulator._default_options()`.
- Regarding `log_level`, in `Terra` and in `Aer`, I only found setting the `log_level` globally as an environment variable, so I am not sure what we can do here.
- `job_tags` - I added to `FakeRuntimeJob`.